admin/modules/forum/attachments.php

```diff
@@ -668,20 +668,25 @@ function() {
 		}
 
 		// LESS THAN or GREATER THAN
+		$direction_fields = array(
+			"dateuploaded" => $mybb->get_input('dateuploaded', 1),
```

Comment:
Oh, forgot to mention one last little thing.. - we decided to go with constants for the 2nd parameter: https://github.com/mybb/mybb/issues/1736 and they're used everywhere, so should be here too. Then it can be merged since there are no other issues.